### PR TITLE
Ce/hide perf

### PR DESCRIPTION
**Pull request by calellowitz**
@Rohit25negi @emord cc: @pr33thi 
The performance report is not supposed to be released until the 15th of the month, this makes it official

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -742,6 +743,9 @@ def post(self, request, *args, **kwargs):
         if indicator == AWW_INCENTIVE_REPORT:
             if not sql_location or sql_location.location_type_name != LocationTypes.BLOCK:
                 return HttpResponseBadRequest()
+            today = datetime.now(timezone('Asia/Kolkata'))
```
Comment (emord) [inline custom/icds_reports/views.py]: Thoughts on standardizing this:

https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/const.py#L10
https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/__init__.py#L67
https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/__init__.py#L778

https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/templatetags/icds_format.py#L102-L103

File: custom/icds_reports/static/js/directives/download/download.directive.js [hunk 2]
```diff
@@ -333,10 +333,17 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
 
     vm.onSelectYear = function (item) {
         if (item.id === new Date().getFullYear()) {
-            vm.months = _.filter(vm.monthsCopy, function(month) {
-                return month.id <= new Date().getMonth() + 1;
-            });
-            vm.selectedMonth = vm.selectedMonth <= new Date().getMonth() + 1 ? vm.selectedMonth : new Date().getMonth() + 1;
+            if (vm.isIncentiveReportSelected() && new Date().getDate() < 15) {
```
Comment (emord) [inline custom/icds_reports/static/js/directives/download/download.directive.js]: Does this code do the following:

If its before the 15th, show months before the previous month. If its the 15th or later, show all months including the previous month?

Comment (calellowitz) [reply custom/icds_reports/static/js/directives/download/download.directive.js]: Close, except replace previous month with current month

Comment (calellowitz) [reply custom/icds_reports/static/js/directives/download/download.directive.js]: Ah, i now realize that it should do what you are describing, although does what i am describing. Will fix

**Comment by calellowitz** [comment]
@emord i finally fixed this pr, ready for a second review

**Review by emord — APPROVED** [review]

**Pull request merged**